geomesa-blobstore/geomesa-blobstore-api/src/main/java/org/locationtech/geomesa/blob/api/Blob.java

```diff
@@ -0,0 +1,41 @@
+package org.locationtech.geomesa.blob.api;
+
+
+public class Blob {
+
+    private String id;
+    private String localName;
+    private byte[] payload;
+
+    public Blob(){
+
+    }
+
+    public Blob(String newID, String newLocalName, byte[] newPayload) {
+        id = newID;
+        localName = newLocalName;
+        payload = newPayload;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getLocalName() {
+        return localName;
+    }
+
+    public byte[] getPayload() {
+        return payload;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I like using Guava's `MoreObjects.toStringHelper()` to get consistent string representations.